Makefile

```diff
@@ -96,7 +90,7 @@ docker_run:
 		-e GOOGLE_APPLICATION_CREDENTIALS=${CREDENTIALS_PATH} \
 		-v $(CURDIR):/cft/workdir \
 		${DOCKER_REPO_BASE_KITCHEN_TERRAFORM} \
-		/bin/bash -c "source test/ci_integration.sh && setup_environment && exec /bin/bash"
+		/bin/bash -c "cd /cft/workdir && source test/ci_integration.sh && setup_environment && exec /bin/bash"
```

Comment:
don't need to cd /cft/workdir, docker will do that automatically as long it's WORK DIR of the container

Reply:
Does it break functionality?

Reply:
I think it will not hurt

Reply:
> Does it break functionality?

if leaving as is doesn't brake either we should probably stick to the least changes